cloud_AWS/terraform/module/examples/multiple-vpc-regions/aws_multiregions.py

```diff
@@ -0,0 +1,174 @@
+import argparse
+import logging
+import json
+import os
+import sys
+from typing import Any, Callable, Dict, Tuple
+
+from python_terraform import IsFlagged, IsNotFlagged, Terraform
+
+log = logging.getLogger(__name__)
+logging.basicConfig(
+    format="%(asctime)s [%(name)s] %(levelname)s: %(message)s", level=logging.INFO, filename="multiregions.log"
+)
+
+EX_OK: int = 0  # exit code for successful command
+EX_FAILED: int = 1  # exit  code for failed command
+
+DFAULT_VPC_REGIONS_FILE_NAME: str = "input_data.json"
+
+TerraformVars = Dict[str, Any]  # variables passed in terraform plan/apply/destroy call
+TerraformAction = Callable[[Terraform, TerraformVars], bool]  # terraform plan/apply/destroy
+
+
+def execute_action(action: TerraformAction, vpc_regions: dict) -> bool:
```

Comment:
Probably mypy would recommend Dict in place of dict

Reply:
Implemented suggestion

Reply:
Resolved